### PR TITLE
add info about DockerHub and tags of images

### PR DESCRIPTION
**Pull request by Avvessalom**
Add info about our DockerHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +29,27 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
...
+docker pull jetbrains/projector-rider:latest
+docker pull jetbrains/projector-rubymine:latest
+docker pull jetbrains/projector-pycharm-p:latest
+docker pull jetbrains/projector-pycharm-c:latest
+docker pull jetbrains/projector-phpstorm:latest
+docker pull jetbrains/projector-idea-u:latest
+docker pull jetbrains/projector-idea-c:latest
+docker pull jetbrains/projector-goland:latest
+docker pull jetbrains/projector-clion:latest
+docker pull jetbrains/projector-datagrip:latest
```
Comment (SerVB) [inline README.md]: Two proposals:
1. Isn't `:latest` redundant?
2. Let's sort the list (like the Space list; it should be more user-friendly).

**Comment by SerVB** [comment]
Oleg, could you please review first please? And I will review after you.

**Comment by SerVB** [comment]
Please request my review before merging. For now I'm going out of this PR to clean the list of pending PRs a bit :)

**Review by olegchir — APPROVED** [review]
Too many grammatical errors. Do you have MSWord or https://languagetool.org/, or any other checker? Also, the text can be much more informative

**Comment by Avvessalom** [comment]
Yes, I have “language tools” and “Grammarly”. Both of these text checkers do not see grammar errors. Could you please point out grammatical errors so that I can correct them?

> Also, the text can be much more informative

This is just new information about versioning our images. What information would you like to see here more?

**Review by olegchir — APPROVED** [review]
Nice to have at least something :)

I think that all these texts should be checked by someone from JB Proofreading Team. But that's another story.

**Comment by cqupt-zjs** [comment]
您好，俺已经收到了恁里邮件，谢谢哈~

File: README.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ Some scripts to create and run a Docker container with Projector and JetBrains I
 
 How to run JetBrains IDE in Docker and access it via a web browser?
 
-Firstly, pull an image with needed IDE:
+Firstly, pull an image with needed IDE 
+
+from Space:
```
Comment (SerVB) [inline README.md]: For now, I find formatting here a bit difficult to understand. Maybe we can add points to make it more clear, like the following:
```suggestion
#### **Option A**. From Space:
```

File: README.md [hunk 3]
```diff
@@ -27,6 +29,29 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
 docker pull registry.jetbrains.team/p/prj/containers/projector-webstorm
 ```
 
+on space published only latest tested IDE and develop version of projector,
+
+or from [DockerHub](https://hub.docker.com/orgs/jetbrains/repositories):
```
Comment (SerVB) [inline README.md]:
```suggestion
#### **Option B**. From [DockerHub](https://hub.docker.com/orgs/jetbrains/repositories):
```

File: README.md [hunk 4]
```diff
@@ -11,7 +11,9 @@ Some scripts to create and run a Docker container with Projector and JetBrains I
 
 How to run JetBrains IDE in Docker and access it via a web browser?
 
-Firstly, pull an image with needed IDE:
+Firstly, pull an image with needed IDE
```
Comment (SerVB) [inline README.md]:
```suggestion
### Step 1

Firstly, pull an image with the needed IDE:
```

File: README.md [hunk 5]
```diff
@@ -27,6 +29,29 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
 docker pull registry.jetbrains.team/p/prj/containers/projector-webstorm
 ```
 
+on space published only latest tested IDE and develop version of projector,
```
Comment (SerVB) [inline README.md]:
```suggestion
On Space, only the latest tested IDE and the `develop` version of Projector are published.
```

File: README.md [hunk 6]
```diff
@@ -27,6 +29,29 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
...
+docker pull jetbrains/projector-goland:latest
+docker pull jetbrains/projector-clion:latest
+docker pull jetbrains/projector-datagrip:latest
+```
+Tags are distributed on DockerHub as follows:
+ - `:latest` This image contains the latest up-to-date version of the tested IDE with the latest stable version of the Projector;
+ - `:develop` This image contains the latest up-to-date version of the tested IDE with the latest commit of the Projector repository; 
+ - `:<IDE_VERSION>-develop` This image contains the selected version of IDE with the latest commit of the Projector repository; 
+ - `<IDE_VERSION>-projector-<PROJECTOR_VERSION>` This image contains the selected version of IDE with the selected stable version of Projector.
+
```
Comment (SerVB) [inline README.md:56]:
```suggestion

### Step 2
```

File: README.md [hunk 7]
```diff
@@ -27,6 +29,29 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
...
+docker pull jetbrains/projector-idea-c:latest
+docker pull jetbrains/projector-goland:latest
+docker pull jetbrains/projector-clion:latest
+docker pull jetbrains/projector-datagrip:latest
+```
+Tags are distributed on DockerHub as follows:
+ - `:latest` This image contains the latest up-to-date version of the tested IDE with the latest stable version of the Projector;
+ - `:develop` This image contains the latest up-to-date version of the tested IDE with the latest commit of the Projector repository; 
+ - `:<IDE_VERSION>-develop` This image contains the selected version of IDE with the latest commit of the Projector repository; 
+ - `<IDE_VERSION>-projector-<PROJECTOR_VERSION>` This image contains the selected version of IDE with the selected stable version of Projector.
```
Comment (SerVB) [inline README.md]:
```suggestion
 - `:latest` – this image contains the **latest up-to-date** version of the tested IDE with the latest **stable** version of the Projector.
 - `:develop` – this image contains the **latest up-to-date** version of the tested IDE with the **latest commit** of the Projector repository.
 - `:<IDE_VERSION>-develop` – this image contains the **selected** version of the IDE with the **latest commit** of the Projector repository. 
 - `<IDE_VERSION>-projector-<PROJECTOR_VERSION>` – this image contains the **selected** version of the IDE with the **selected stable** version of Projector.
```

File: README.md (hunk 5 above)
Comment (SerVB) [reply README.md]: By the way, at this point, the reader doesn't know yet what is `develop`. Maybe swapping these two options (Space and Docker Hub)?

File: README.md (hunk 7 above)
Comment (SerVB) [reply README.md]: Maybe a table will be better here, you can try to check it if you want

**Review by SerVB — CHANGES_REQUESTED** [review]
This can be improved in terms of formatting, grammar, and user-friendliness.

File: README.md (hunk 2 above)
Comment (Avvessalom) [reply README.md]: Good note. Indeed, it doesn't seem very clear when looking at it for the first time. I found an elegant solution. It also slightly changes the structure of the image description to a more understandable one.

File: README.md [hunk 8]
```diff
@@ -11,7 +11,32 @@ Some scripts to create and run a Docker container with Projector and JetBrains I
...
+docker pull jetbrains/projector-pycharm-p
+docker pull jetbrains/projector-rider
+docker pull jetbrains/projector-rubymine
+docker pull jetbrains/projector-webstorm
+```
+Tags are distributed on DockerHub as follows:
+- `:latest` – this image contains the **latest up-to-date** version of the tested IDE with the latest **stable** version of the Projector;
+- `:develop` – this image contains the **latest up-to-date** version of the tested IDE with the **latest commit** of the Projector repository;
+- `:<IDE_VERSION>-develop` – this image contains the **selected** version of the IDE with the **latest commit** of the Projector repository;
+- `<IDE_VERSION>-projector-<PROJECTOR_VERSION>` – this image contains the **selected** version of the IDE with the **selected stable** version of Projector.
```
Comment (SerVB) [inline README.md]: Just noticed this little guy missing :)

```suggestion
- `:<IDE_VERSION>-projector-<PROJECTOR_VERSION>` – this image contains the **selected** version of the IDE with the **selected stable** version of Projector.
```

File: README.md [hunk 9]
```diff
@@ -27,6 +52,8 @@ docker pull registry.jetbrains.team/p/prj/containers/projector-rubymine
 docker pull registry.jetbrains.team/p/prj/containers/projector-webstorm
 ```
 
+### Step 2
+
 After that, you can run it via the following command (just replace `IMAGE_NAME` with the needed name, for
 example, `registry.jetbrains.team/p/prj/containers/projector-clion`):
```
Comment (SerVB) [inline README.md]: Just reread the readme and see that the text here can be simplified a bit:

```suggestion
example, `jetbrains/projector-clion`):
```

**Review by SerVB — APPROVED** [review]
LGTM, but a couple of minor suggestions

File: README.md (hunk 9 above)
Comment (Avvessalom) [reply README.md]: in the original README the path to our Space repository.
ok, I will replace Space with DockerHub

**Pull request merged**